### PR TITLE
add a member variable in TFGraphHolder

### PR DESCRIPTION
**Pull request by chengshunzhang**
Add a member val inDriver in class TFGraphHolder in object TFNet so that we do not need to repeatedly call isDriver when on worker node. Test using ssd_mobilenet_v1_coco with no problems.

### CODE REVIEW

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/net/TFNet.scala [hunk 1]
```diff
@@ -253,6 +253,9 @@ object TFNet {
   class TFGraphHolder(@transient var tfGraph: Graph, private var id: String)
     extends Serializable with KryoSerializable {
 
+    @transient
+    lazy val isDr = isDriver
```
Comment (yangw1234) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/net/TFNet.scala]: change the variable name to a more meaningful one?

**Comment by yangw1234** [comment]
please add more description about what does this pr do and describe your test result

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/net/TFNet.scala [hunk 2]
```diff
@@ -254,7 +254,7 @@ object TFNet {
     extends Serializable with KryoSerializable {
 
     @transient
-    lazy val isDr = isDriver
+    lazy val theDriver = isDriver
```
Comment (yangw1234) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/net/TFNet.scala]: change to inDriver?

Comment (chengshunzhang) [reply zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/net/TFNet.scala]: OK.

**Comment by yangw1234** [comment]
LGTM

**Comment by yangw1234** [comment]
pr validation pass

**Pull request merged**